recipes-extend/target-readme/files/images/dtbs/overlays/Readme.md

```diff
@@ -0,0 +1,24 @@
+# RZ/G2L-SBC overlays Folder
+
+## Description
+
+This directory includes Device Tree Overlay (DTO) files that extend or modify the base Device Tree configuration. Overlays are used to enable or configure additional hardware features or peripherals on the RZ/G2L-SBC platform.
+
+## A top-level directory of overlays
+
+```
+overlays                                       
+├── rzg2l-sbc-can.dtbo                         <---- Overlay for CAN interface
+├── rzg2l-sbc-dsi.dtbo                         <---- Overlay for DSI display interface
+├── rzg2l-sbc-ext-i2c.dtbo                     <---- Overlay for external I2C devices
+├── rzg2l-sbc-ext-spi.dtbo                     <---- Overlay for external SPI devices
+├── rzg2l-sbc-ov5640.dtbo                      <---- Overlay for OV5640 camera
```

Comment:
Please ensure you have checked the build output before making any changes to the hierarchy of README files, as rzg2l-sbc-*.dtbo does not exist.

Reply:
Update in the commit: fa4a70cd5eb6037a6490416b3944b4b8ce0fae77